### PR TITLE
change cluster_manager, restfulapi and deploy.py to python3

### PR DESCRIPTION
**Pull request by xudifsd**
Most of changes are made by `2to3` program.

Also removed a lot of dead code in deploy.py.

Have tested cluster manager and restful api. Works, but not sure if we will meet some problem.

### CODE REVIEW

File: src/ClusterBootstrap/gs_tools.py [hunk 1]
```diff
@@ -347,13 +347,13 @@ def create_storage_with_config( configGrp, location ):
     storagename = configGrp["name"] + location
     output = create_storage_account( storagename, configGrp["sku"], location)
     if verbose: 
-        print ( "Storage account %s" % output )
+        print(( "Storage account %s" % output ))
```
Comment (Anbang-Hu) [inline src/ClusterBootstrap/gs_tools.py]: double brackets

Comment (xudifsd) [reply src/ClusterBootstrap/gs_tools.py]: Yes, I see many of this. I think it's because 2to3 will add brace whenever it encountered a `print`

Comment (xudifsd) [reply src/ClusterBootstrap/gs_tools.py]: Will delete these

File: src/ClusterBootstrap/az_tools.py [hunk 2]
```diff
@@ -53,15 +53,15 @@ def merge_config(config1, config2, verbose):
         if entry in config1:
             if isinstance(config1[entry], dict):
                 if verbose:
-                    print("Merge entry %s " % entry)
+                    print(("Merge entry %s " % entry))
```
Comment (Anbang-Hu) [inline src/ClusterBootstrap/az_tools.py]: double braces

**Comment by xudifsd** [comment]
after this, we can use https://pypi.org/project/autopep8/ to format all python code

**Pull request merged**